_ont/ont-cig-g-97cp.md

```diff
@@ -1,36 +1,50 @@
-| Form Factor      | ONT              |
- 
-{% include image.html file="cig/cig-1.jpg" alt="CIG G-97CP Unidata" caption="CIG G-97CP Unidata <a href='https://forum.fibra.click/u/papin'>@papin</a>" %}
-{% include image.html file="cig/cig-2.jpg" alt="CIG G-97CP Unidata" caption="CIG G-97CP Unidata <a href='https://forum.fibra.click/u/papin'>@papin</a>" %}
+|                  |                                                                          |
+| ---------------- |--------------------------------------------------------------------------|
+| Vendor/Brand     | CIG                                                                      |
+| Model            | G-97CP                                                                   |
+| ODM              | CIG                                                                      |
+| ODM Product Code | G-97CP                                                                   |
+| ODM              | ✅                                                                        |
+| Chipset          | Realtek RTL9601D                                                         |
+| Flash            |                                                                          |
+| RAM              |                                                                          |
+| System           | Linux                                                                    |
+| 2.5GbaseT        | Yes                                                                      |
+| Optics           | SC/APC                                                                   |
+| IP address       | 192.168.100.1                                                            |
+| Web Gui          |                                                                          |
+| SSH              | ✅ user ```root```, password same as [enable](##Enable-Password) password |
+| Telnet           |                                                                          |
+| Serial           | ✅                                                                        |
+| Form Factor      | ONT                                                                      |
+
+{% include image.html file="cig/cig-1.jpg" alt="CIG G-97CP Unidata" caption="CIG G-97CP
+Unidata <a href='https://forum.fibra.click/u/papin'>@papin</a>" %} {% include image.html file="cig/cig-2.jpg" alt="CIG
+G-97CP Unidata" caption="CIG G-97CP Unidata <a href='https://forum.fibra.click/u/papin'>@papin</a>" %}
+
+## Enable password
+
+{% include alert.html content="The following enable password is used to enter GponCLI, but currently in the models distributed in Italy by TIM and OpenFiber there is no way to enter GponCLI either by SSH, Telnet or Serial, with the exception of Unidata, with the Unidata one you can access the GponCLI by Serial and SSH. The enable password is not useful for entering the Web Gui." alert="Note" icon="svg-info" color="blue" %}
```

Comment:
Change content

Reply:
In the content do you want me to write just the Unidata part ? Because that's what I added basically.

Reply:
yes just unidata part. OF and Tim distributes the f6005.

Reply:
Done, let me know if now its ok. i'll send you the rerevision